inst/Upgrades.qml

```diff
@@ -0,0 +1,147 @@
+import QtQuick		2.12
+import JASP.Module	1.0
+
+Upgrades
+{
+	Upgrade
+	{
+		functionName:	"ClassicalMetaAnalysis"
+		fromVersion:	"0.15"
```

Comment:
given we'll merge this after this release, you can change it to `fromVersion: "0.16.4"` and `toVersion: "0.17"`